src/Autosuggest.js

```diff
@@ -156,6 +159,14 @@ class Autosuggest extends Component {
     }
   }
 
+  maybeSelectFirstSuggestion() {
+    const { selectFirstSuggestion, multiSection, updateFocusedSuggestion } = this.props;
+
+    if (selectFirstSuggestion) {
+      updateFocusedSuggestion(multiSection ? 0 : null, 0);
+    }
+  }
```

Comment:
When `selectFirstSuggestion` is `false`, this function will be called over and over again just to perform the destructuring and the `if` check. If wonder whether it worth to set `maybeSelectFirstSuggestion` to `noop` in the constructor if we know that `selectFirstSuggestion === false` (this is obviously assuming that `selectFirstSuggestion` cannot be changed during the lifetime of an Autosuggest). Do you think it worth the effort?